frontend/src/components/job/info.js

```diff
@@ -23,11 +23,23 @@ class JobInfo extends Component {
                 return null
             })
 
+            let meta = [];
+            let metaTag = '<none>';
+
+            Object.keys(taskGroup.Meta || {}).forEach(function(key) {
+                meta.push(<dt key={key + 'dt'}>{key}</dt>);
+                meta.push(<dd key={key + 'dd'}>{taskGroup.Meta[key]}</dd>);
+            });
+
+            if (meta.length > 0) {
+                metaTag = <dl className="dl-horizontal">{meta}</dl>
+            }
+
             return (
                 <tr key={taskGroup.ID}>
                     <td>{taskGroup.Name}</td>
                     <td>{taskGroup.Count}</td>
-                    <td>{taskGroup.Meta || "<none>" }</td>
+                    <td>{metaTag}</td>
```

Comment:
for me at least, my metadata is used to define auto-scale rules from my external system and other need-to-know information. Having this directly visible is important for me at least, not sure how other people use it. Personally I'm willing to sacrifice screen estate for more data directly visible. Would like to keep it in for now and see feedback on it.

A compromise could be long term to show number of tags as human readable, and abuse `tooltip` on hover to show the actual data :)